cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/PredicateFunctionsTest.scala

```diff
@@ -52,7 +52,8 @@ class PredicateFunctionsTest extends DocumentingTest {
         |* <<functions-single,single()>>""")
     graphViz()
     section("all()", "functions-all") {
-      p("`all()` returns true if the predicate holds for all elements in the given list.")
+      p("`all()` returns true if the predicate holds for all elements in the given list." +
+        " Returns `null` if the list is `null` or all of its elements are `null`.")
```

Comment:
"Returns `null` if the..." -> "`null` is returned if the ..."  (to make it a sentence, rather than a clause)

Reply:
There is something here in terms of tabularizing this somehow (when it would revert to "Returns null if...") - but that's for another day. For now, as it is a textual in-situ sentence, this will work.